wodles/aws/constants.py

```diff
@@ -0,0 +1,159 @@
+    'me-central-1', 'me-south-1', 'sa-east-1', 'us-east-1', 'us-east-2', 'us-west-1', 'us-west-2'
+)
+DEFAULT_AWS_INTEGRATION_GOV_REGIONS = {'us-gov-east-1', 'us-gov-west-1'}
+SERVICES_REQUIRING_REGION = {'inspector', 'cloudwatchlogs'}
+
+
+# URLS
+RETRY_CONFIGURATION_URL = 'https://documentation.wazuh.com/current/amazon/services/prerequisites/' \
+                          'considerations.html#Connection-configuration-for-retries'
+GUARDDUTY_URL = 'https://documentation.wazuh.com/current/amazon/services/supported-services/guardduty.html'
+AWS_CREDENTIALS_URL = 'https://documentation.wazuh.com/current/amazon/services/prerequisites/credentials.html'
+SECURITY_LAKE_IAM_ROLE_AUTHENTICATION_URL = 'https://documentation.wazuh.com/current/cloud-security/amazon/services/' \
+                                        'supported-services/security-lake.html#configuring-an-iam-role'
+
+# AWS INTEGRATION DEPRECATED VALUES
+DEPRECATED_AWS_INTEGRATION_TABLES = {'log_progress', 'trail_progress'}
+
+# DEPRECATED MESSAGES
+GUARDDUTY_DEPRECATED_MESSAGE = 'The functionality to process GuardDuty logs stored in S3 via Kinesis was deprecated ' \
+                               'in {release}. Consider configuring GuardDuty to store its findings directly in an S3 ' \
+                               'bucket instead. Check {url} for more information.'
+
+AWS_AUTH_DEPRECATED_MESSAGE = 'The {name} authentication parameter was deprecated in {release}. ' \
+                     'Please use another authentication method instead. Check {url} for more information.'
+
+
+# ERROR CODES
+INVALID_CREDENTIALS_ERROR_CODE = "SignatureDoesNotMatch"
+INVALID_REQUEST_TIME_ERROR_CODE = "RequestTimeTooSkewed"
+THROTTLING_EXCEPTION_ERROR_CODE = "ThrottlingException"
+UNKNOWN_ERROR_CODE = 1
+INVALID_CREDENTIALS_ERROR_CODE = 3
+METADATA_ERROR_CODE = 5
+UNABLE_TO_CREATE_DB = 6
+UNEXPECTED_ERROR_WORKING_WITH_S3 = 7
+DECOMPRESS_FILE_ERROR_CODE = 8
+PARSE_FILE_ERROR_CODE = 9
+UNABLE_TO_CONNECT_SOCKET_ERROR_CODE = 11
+INVALID_TYPE_ERROR_CODE = 12
+SENDING_MESSAGE_SOCKET_ERROR_CODE = 13
+EMPTY_BUCKET_ERROR_CODE = 14
+INVALID_ENDPOINT_ERROR_CODE = 15
+THROTTLING_ERROR_CODE = 16
+INVALID_KEY_FORMAT_ERROR_CODE = 17
+INVALID_PREFIX_ERROR_CODE = 18
+INVALID_REQUEST_TIME_ERROR_CODE = 19
+UNABLE_TO_FETCH_DELETE_FROM_QUEUE = 21
+INVALID_REGION_ERROR_CODE = 22
+
+
+# ERROR MESSAGES
+INVALID_CREDENTIALS_ERROR_MESSAGE = "Invalid credentials to access S3 Bucket"
+INVALID_REQUEST_TIME_ERROR_MESSAGE = "The server datetime and datetime of the AWS environment differ"
+THROTTLING_EXCEPTION_ERROR_MESSAGE = "The '{name}' request was denied due to request throttling. " \
+                                     "If the problem persists check the following link to learn how to use " \
+                                     f"the Retry configuration to avoid it: '{RETRY_CONFIGURATION_URL}'"
+
+# TESTS VALUES
+WAZUH_VERSION = "4.5.0"
```

Comment:
`WAZUH_VERSION` should have its current value and not a hardcoded one to avoid missunderstandings.

Reply:
This constant `WAZUH_VERSION` is used only for hardcoding the Wazuh Version value when patching some functions. I changed the name to `TEST_HARDCODED_WAZUH_VERSION` so as not to confuse.